scenarios/reproducers/3-nodes.yml

```diff
@@ -14,14 +14,12 @@ cifmw_rhol_crc_config:
 # Since we don't have zuul data, we have to provide some
 # networking information to the libvirt_manager so that it will be able to
 # generate the needed content for later usage
-cifmw_libvirt_manager_networks:
+_networks:
   public:
     range: "192.168.100.0/24"
   osp_trunk:
-    default: true
     range: "192.168.122.0/24"
     mtu: 1500
-    static_ip: true
   internal-api:
```

Comment:
Do we really need to preserve `storage`,  `tenant`,  `storage-mgmt` and `internal-api`? Where are those values consumed?

Reply:
not consumed anymore - more cleaning and converging will be needed as well. I'll probably update doc and related to instruct "include the networking-definition first, then the layout" so that we can inherit parameters the right way.

Reply:
cleaned.